athena_cfn/.taskcat.yml

```diff
@@ -0,0 +1,15 @@
+# https://aws.amazon.com/blogs/infrastructure-and-automation/introducing-taskcat-v0-9/
+project:
+  name: athena-cfn
+  regions:
+  - us-east-1
+  parameters:
+    YourCompany: quilt-examples
```

Comment:
I don't understand this value relative to its name `YourCompany`.

Reply:
We use the company name in the Quilt bucket we create, for uniqueness. TaskCat has to spoof that for testing.